### ISSUE TITLE
Expose more color options

### ISSUE THREAD

**Issue by Charuru** [RocketChat/Rocket.Chat#2347]
How do I expose more color options to the theme settings? For example I want to change the user color:

```
.message {
    &.new-day {
        &:before {
            color: @secondary-font-color;
            background-color: @content-background-color;
        }
        &:after {
            border-top-color: #ddd;
        }
    }
    .user {
        color: #444444;
        &:hover {
            color: #333;
        }
    }
    .info {
        color: @info-font-color;
    }
    &.system {
        .body {
            color: @info-font-color;
        }
    }
    a {
        color: @link-font-color;
        &:hover {
            color: darken(@link-font-color, 10%);
        }
    }
}

```

without changing the css directly.

**Comment by rodrigok** [comment]
We need to refactor our css/theme, so we can't do it right now.

Now there is a new field to paste custom CSS/LESS in admin page, probably you can solve your problem by now

**Comment by timkinnane** [comment]
+1 on this. I've just got half-way through trying to create a custom theme using the exposed variables, but found there are so many hard-coded colours in **base.less** that its near impossible to deviate from the standard theme. e.g. You really can't have a light coloured background for the side-nav, because there's too many elements with light text colours manually set.

It would be quite an effort, but really needs someone to go through base.less and change most of the inline hex codes to be variable references for the defined colour settings. I'm happy to do it but it will be quite a while before I have the time.

**Comment by engelgabriel** [comment]
Fixed by #4803
